module-cheatsheets/pathway-analysis-cheatsheet.md

```diff
@@ -0,0 +1,83 @@
+| `DESeq2`                | [`lfcShrink()`](https://rdrr.io/bioc/DESeq2/man/lfcShrink.html)            | Shrink Log Fold Changes                                        | Adds shrunken log2 fold changes to the results of a `DESeqDataSet` object                         |
+| `DESeq2`                | [`resultsNames()`](https://www.rdocumentation.org/packages/DESeq2/versions/1.12.3/topics/results)            | Results Names                                        | Returns the names of the estimated effects or coefficients of the `DESeq` model                         |
+
+### `enrichplot`
+
+Read the `enrichplot` package documentation [**here**](https://bioconductor.org/packages/devel/bioc/manuals/enrichplot/man/enrichplot.pdf).
+
+| Library/Package               | Piece of Code                                                 | What it's called      | What it does                                                             |
+|-------------------------------|--------------------------------------------------------------|--------------------------------|--------------------------------------------------------------------------|
+| `enrichplot`                | [`dotplot()`](https://yulab-smu.github.io/clusterProfiler-book/chapter12.html#dot-plot)             | Dot plot                                         | Produces a dot plot for given enrichment results                         |
+| `enrichplot`                | [`upsetplot()`](https://yulab-smu.github.io/clusterProfiler-book/chapter12.html#upset-plot)             | Upset plot                                       | Produces an upset plot, which shows the overlapping genes between gene sets, for given enrichment results                      |
+| `enrichplot`                | [`gseaplot()`](https://yulab-smu.github.io/clusterProfiler-book/chapter12.html#running-score-and-preranked-list-of-gsea-result)             | GSEA plot                                       | Produces a plot visualization displaying the distribution of gene set and enrichment score                      |
+
+### `clusterProfiler`
+
+Read the `clusterProfiler` package documentation [**here**](https://www.rdocumentation.org/packages/clusterProfiler/versions/3.0.4).
+
+| Library/Package               | Piece of Code                                                 | What it's called      | What it does                                                             |
+|-------------------------------|--------------------------------------------------------------|--------------------------------|--------------------------------------------------------------------------|
+| `clusterProfiler`                | [`GSEA()`](https://www.rdocumentation.org/packages/clusterProfiler/versions/3.0.4/topics/GSEA)             | Gene Set Enrichment Analysis (GSEA)                                         | Performs a universal gene set enrichment analysis on given set of gene sets                       |
```

Comment:
What do the gene sets provided need to look like in R object terms?

Reply:
I'll put more detail around that 👍